source/OutfitterPanel.cpp

```diff
@@ -930,24 +959,28 @@ void OutfitterPanel::CheckRefill()
 			int amount = ship->Attributes().CanAdd(*outfit, numeric_limits<int>::max());
 			if(amount > 0)
 			{
-				bool available = outfitter.Has(outfit) || player.Stock(outfit) > 0;
-				available = available || player.Cargo().Get(outfit) || player.Storage().Get(outfit);
-				if(available)
-					needed[outfit] += amount;
+				bool isAvailable = outfitter.Has(outfit) || player.Stock(outfit) > 0;
+				isAvailable = isAvailable || player.Cargo().Get(outfit) || player.Storage().Get(outfit);
```

Comment:
```suggestion
				bool isAvailable = outfitter.Has(outfit) || player.Stock(outfit) > 0
					|| player.Cargo().Get(outfit) || player.Storage().Get(outfit);
```
If you're changing it anyway, you can join these two expressions.